### PR TITLE
Add xvec, dask-geopandas, contextily

### PR DESCRIPTION
**Pull request by scottyhq**
* https://github.com/xarray-contrib/xvec
* https://contextily.readthedocs.io/en/latest/
* https://github.com/geopandas/dask-geopandas

### CODE REVIEW

File: pangeo-notebook/packages.txt [hunk 1]
```diff
@@ -117,6 +118,7 @@ dask==2024.6.2
 dask-core==2024.6.2
 dask-expr==1.1.6
 dask-gateway==2024.1.0
+dask-geopandas==0.3.1
```
Comment (weiji14) [inline pangeo-notebook/packages.txt:121]: Should we try relocking to get `dask-geopandas=0.4.1` (https://github.com/conda-forge/dask-geopandas-feedstock/pull/17)? Or do you want a version tag with 0.3.1 (given all the changes with geopandas 1.0 dropping pygeos and supporting shapely 2.0+ and all that).

Comment (scottyhq) [reply pangeo-notebook/packages.txt:121]: /condalock

File: ml-notebook/packages.txt [hunk 2]
```diff
@@ -205,15 +189,15 @@ fribidi==1.0.10
...
 gcc_impl_linux-64==12.3.0
 gcc_linux-64==12.3.0
 gcm_filters==0.3.0
 gcsfs==2024.6.0
 gdal==3.9.0
 gdk-pixbuf==2.42.12
-geocube==0.5.2
+geocube==0.5.3
 geographiclib==2.0
 geopandas==0.14.4
```
Comment (scottyhq) [inline ml-notebook/packages.txt:202]: still not 1.0, so maybe some other package is pinning <1

Comment (weiji14) [reply ml-notebook/packages.txt:202]: Tracking issue at https://github.com/conda-forge/geocube-feedstock/pull/40

File: pangeo-notebook/packages.txt [hunk 3]
```diff
@@ -177,20 +177,20 @@ gcc_impl_linux-64==12.3.0
...
 gcsfs==2024.6.0
-gdal==3.9.0
+gdal==3.9.1
 gdk-pixbuf==2.42.12
-geocube==0.5.2
+geocube==0.6.0
 geographiclib==2.0
-geopandas==0.14.4
-geopandas-base==0.14.4
+geopandas==1.0.0
```
Comment (scottyhq) [inline pangeo-notebook/packages.txt:184]: :rocket:

**Review by weiji14 — APPROVED** [review]
LGTM!

**Pull request merged**